### DISCUSSION TITLE
Osu!Lazer drivers

### DISCUSSION THREAD

**Discussion by Reachify200** [ppy/osu#21874] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Hi, so I'm better on using my wacom drivers for my wacom one, but whenever I launch osu lazer, my cursor get's stuck in the right corner of the screen, only letting me move it up and down. I don't wanna use the in-game tablet drivers as I use a weird portion for my tablet, but whenever I turn the osu drivers on that's when my tablet works. Please help me or tell me if there is any way to not use the in-game drivers.

### Screenshots or videos

https://user-images.githubusercontent.com/121544608/209784380-c328de7a-f3a4-49be-aee2-b8d167957c3d.mp4

### Version

2022.1227.0

### Logs

[runtime.log](https://github.com/ppy/osu/files/10313153/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10313154/updater.log)
[database.log](https://github.com/ppy/osu/files/10313155/database.log)
[input.log](https://github.com/ppy/osu/files/10313156/input.log)
[network.log](https://github.com/ppy/osu/files/10313157/network.log)
[performance.log](https://github.com/ppy/osu/files/10313158/performance.log)
.log)

**Comment by bdach** [comment, marked as answer]
I'm deeply confused. Your video shows the "tablet" handler _disabled_ in settings. Does that not fix your issue?

If it does not, please try disabling "High precision mouse".

**Reply by Reachify200** [reply]
I don't wanna use the tablet driver or whatever it is from osu lazer, as it has a complicated ui, and where can I find high precision mouse?

**Reply by bdach** [reply]
It's in mouse settings.

**Reply by Reachify200** [reply]
I already have that turned off

**Reply by bdach** [reply]
If you have both tablet handler off and high precision mouse off then I don't know. Windows Ink enabled maybe?

**Reply by Reachify200** [reply]
that worked! thank you so much!
